components/dashboard/src/settings/Integrations.tsx

```diff
@@ -394,7 +393,7 @@ function GitIntegrations() {
         </div>
 
         {providers && providers.length === 0 && (
-            <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100">
+            <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100 dark:bg-gray-900">
                 <div className="m-auto text-center">
                     <h3 className="self-center text-gray-500 mb-4">No Git Integrations</h3>
```

Comment:
**issue**: This will also need a `text-gray-400` on dark theme as all `h3` headings on empty state components.